AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage.py

```diff
@@ -5,47 +5,52 @@
-import scripting_utils.scripting_tools as scripting_tools
-import azlmbr.legacy.general as general
-import azlmbr.paths as paths
-from scripting_utils.scripting_constants import (WAIT_TIME_3, BASE_LEVEL_NAME)
-
-EXPECTED_LINES = ["Activator Script: Activated", "Deactivator Script: Deactivated"]
-controller_dict = {
-    "name": "Controller",
-    "status": "active",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "controller.scriptcanvas"),
-}
-activated_dict = {
-    "name": "ActivationTest",
-    "status": "inactive",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "activator.scriptcanvas"),
-}
-deactivated_dict = {
-    "name": "DeactivationTest",
-    "status": "active",
-    "path": os.path.join(paths.projectroot, "ScriptCanvas", "OnEntityActivatedScripts", "deactivator.scriptcanvas"),
-}
-ENTITY_TO_ACTIVATE_PATH = "Configuration|Properties|Variables|EntityToActivate|Datum|Datum|value|EntityToActivate"
-ENTITY_TO_DEACTIVATE_PATH = "Configuration|Properties|Variables|EntityToDeactivate|Datum|Datum|value|EntityToDeactivate"
-WAIT_ONE_SECOND = 1.0
+
 # fmt: off
 class Tests():
-    controller_exists    = ("Successfully found controller entity",  "Failed to find controller entity")
-    activated_exists     = ("Successfully found activated entity",   "Failed to find activated entity")
-    deactivated_exists   = ("Successfully found deactivated entity", "Failed to find deactivated entity")
-    start_states_correct = ("Start states set up successfully",      "Start states set up incorrectly")
     game_mode_entered    = ("Successfully entered game mode",         "Failed to enter game mode")
     lines_found          = ("Successfully found expected prints",    "Failed to find expected prints")
     game_mode_exited     = ("Successfully exited game mode", "Failed to exit game mode")
 # fmt: on
 
-class ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage():
+
+def setup_level_entities(entity_dictionary):
+    """
+    Helper function for setting up the 3 entities and their script canvas components needed for this test
+
+    """
+    import azlmbr.math as math
+    import scripting_utils.scripting_tools as scripting_tools
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.editor_component.editor_script_canvas import ScriptCanvasComponent
+    from editor_python_test_tools.editor_component.editor_script_canvas import VariableState
+    from editor_python_test_tools.utils import TestHelper as TestHelper
+    from scripting_utils.scripting_constants import (WAIT_TIME_3)
+
+    position = math.Vector3(512.0, 512.0, 32.0)
+
+    #create entity, give it a script canvas component and set its active status
+    editor_entity = EditorEntity.create_editor_entity_at(position, entity_dictionary["name"])
+    editor_entity_sc_component = ScriptCanvasComponent(editor_entity)
```

Comment:
rename to `scriptcanvas_component`

Reply:
I'm going to rename this in all the scripts related to this PR